package.json

```diff
@@ -10,10 +10,14 @@
     "babel-core": "^6.2.1",
     "babel-preset-es2015": "^6.1.18",
     "babel-register": "^6.2.0",
-    "mocha": "^2.2.5"
+    "mocha": "^2.2.5",
+    "rimraf": "^2.5.0"
   },
   "scripts": {
+    "clean": "rimraf lib",
```

Comment:
It fails if the directory does not exist (first run), and won't work on Windows.